apps/accounts/views/password_reset.py

```diff
@@ -39,6 +41,7 @@ def reset_password(request):
         return HttpResponseRedirect(reverse('home'))
 
 
+@waffle_flag('login-global')
```

Comment:
would placing a flag link this on a route in a `urls.py` file also work?

Reply:
That works in the urls.py and I like that better! Will update to do it that way. TY!

Reply:
I changed to use waffle switches instead of flags. A switch is simpler for these use cases. I left the custom waffle flag in place for future use.
I moved the logic to the routes in urls.py, which makes it more readable.